packages/video_player/video_player_avfoundation/darwin/video_player_avfoundation/Sources/video_player_avfoundation/include/video_player_avfoundation/FVPVideoPlayerPlugin_Test.h

```diff
@@ -25,22 +16,6 @@
 
 #pragma mark -
 
-// TODO(stuartmorgan): Move this whole class to its own files.
-@interface FVPVideoPlayer : NSObject <FlutterStreamHandler, FlutterTexture>
-@property(readonly, nonatomic) AVPlayer *player;
```

Comment:
@stuartmorgan Please have a look at [this commit](https://github.com/flutter/packages/pull/8171/commits/717315e53ead6125174e5da3bb9210f55a5b8d12) and tell whether this TODO was resolved by it. I moved `FVPAVFactory` protocol to a separate file, and unified this `FVPVideoPlayer` with the other one I just created, so now everything is in `FVPVideoPlayer.h` file. I'm not sure this is the desired way, though, I found this file (`FVPVideoPlayerPlugin_Test.h`) and its content a little bit confusing.

Reply:
`*_Test.h` headers are where we declare things that would otherwise be declared in the implementation file, but we need to access them in tests. It's a convention that is basically analogous to Dart's `@visibleForTesting`.

Now that `FVPVideoPlayer` is in its own file, it should—since it has things that only need to be exposed for tests: `playerLayer`, `onTextureUnregistered:`, maybe `FlutterStreamHandler` and/or `FlutterTexture` conformance (I didn't check where those are used)—have its own `_Test` header that declared (via a private category; see `FVPVideoPlayerPlugin` below) its visible-only-to-test properties and methods.

Reply:
Okay, thanks. I added a separate `_Test.h` file then, with a test interface for the player